src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Figure implements Drawing, AreaFigure {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public void draw() {
+        String message = "Figure";
+        System.out.println(message);
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }
```

Comment:
```suggestion
```

Reply:
It is enough to make the class abstract and implement methods in each necessary figure, without the default implementation here.